frontend/src/styles/_uswds-theme-custom-styles.scss

```
@@ -21,3 +21,9 @@ i.e.
 */
 
 @use "uswds-core" as *;
+
+.pdf-card {
+  &:hover {
+    transform: scale(1.04);
+  }
+}
```

```suggestion
.pdf-card {
  transition: transform 0.2s;
  &:hover {
    transform: scale(1.04);
  }
}
```
^ A lil' subtle flair.